subprojects/docs/src/docs/release/notes.md

```diff
@@ -53,6 +53,12 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+### Using custom local build cache implementations
```

Comment:
This also needs to go into the upgrade notes.

Reply:
Mentioned.

Reply:
Do we have anything to recommend to people which have been using a custom local build cache? Don't do it, or shall they use the custom local cache as a remote cache?

Reply:
I'd expect very few people have ever tried to use a custom local cache, and those should be the more tech savvy ones. I think it's okay to mention as much as we do already.